src/tests/frontend/specs/responsiveness.js

```diff
@@ -1,91 +1,56 @@
 'use strict';
 
-// Test for https://github.com/ether/etherpad-lite/issues/1763
-
-// This test fails in Opera, IE and Safari
-// Opera fails due to a weird way of handling the order of execution,
-// yet actual performance seems fine
-// Safari fails due the delay being too great yet the actual performance seems fine
-// Firefox might panic that the script is taking too long so will fail
-// IE will fail due to running out of memory as it can't fit 2M chars in memory.
-
-// Just FYI Google Docs crashes on large docs whilst trying to Save,
-// it's likely the limitations we are
-// experiencing are more to do with browser limitations than improper implementation.
-// A ueber fix for this would be to have a separate lower cpu priority
-// thread that handles operations that aren't
-// visible to the user.
+// Wait helper function, for simulating words per minute.
+const wait = (milliseconds) => new Promise((resolve) => setTimeout(resolve, milliseconds));
 
-// Adapted from John McLear's original test case.
-
-xdescribe('Responsiveness of Editor', function () {
+// Test for https://github.com/ether/etherpad-lite/issues/1763
+describe('Responsiveness of Editor', function () {
   // create a new pad before each test run
   beforeEach(function (cb) {
-    helper.newPad(cb);
-    this.timeout(6000);
+    helper.newPad(cb, 'TEST_PAD_collab');
+    this.timeout(1200000);
```

Comment:
Is 20 minutes intentional? Or did you mean 2 minutes?

Reply:
Intentional.  Takes a while to run!